recipes/flake8-pyi/LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
Please don't remove the files, rebase on master and they should be gone.

Reply:
this is not my recipes, am I seeing this by accident?

Reply:
You have probably deleted this while developing, just rebase on master to fix this.

Reply:
what is flake8-pyi? I am not familiar with it... my code is 100% C, no Python

Reply:
It was another recipe that had been added to conda-forge (#10539). I guess you saw it when preparing your PR and deleted it.

Reply:
It's another recipe that you deleted in https://github.com/conda-forge/staged-recipes/pull/10543/commits/fd4ede18f3f853cf07a3a58bb1a291bb10712b65.
You can remove that commit with 
```sh
git rebase fd4ede1 --onto fd4ede1^
git push -f
```

Reply:
Hmm... I now realize my mistake. I deleted all the other directories in my branch except for my own and "examples". I really hope that didn't cause any damage... anyway, I did some pulling, pushing, rebasing and now my branch says "This branch is 102 commits ahead of conda-forge:master.". Is that ok?

Reply:
The last merge commit wasn't right.
```
git reset HEAD~1
git push -f
```
should fix it.

Reply:
Hi guys, I can't figure how to fix this... but I see that the files I deleted were deleted anyway (see below). What should I do? I am thinking that maybe the easiest would be to just abandon this fork and start a new one?

commit 0085150388191bd73b97f3bfed0847a7607f58a4
Author: Travis-CI on github.com/conda-forge/staged-recipes <conda-forge@googlegroups.com>
Date:   Wed Jan 8 14:42:24 2020 +0000

Removed recipes (flake8-pyi, omegaconf) after converting into feedstocks. [ci skip]
delete mode 100644 recipes/flake8-pyi/LICENSE
 delete mode 100644 recipes/flake8-pyi/meta.yaml
 delete mode 100644 recipes/omegaconf/meta.yaml